### DISCUSSION TITLE
Prescribed loss rates for radiatively active tracers in ghg_mam4 chemistry

### DISCUSSION THREAD

**Discussion by dan800** [NCAR/amwg_dev#272] [TF1: Unified physics for L58/L93]
**Q: _Are the prescribed loss rates (from circa 1997) still adequate for the task of getting the GHGs and LW cooling rates right in CAM7? Is there an argument for doing a CAM7-CHEM MT run (or perhaps a WACCM6 run) and saving them out?_** 

**Background:**

The GHG loss rates for H2O, CH4, N2O, CFC11 and CFC12 used in mo_ghg_chem.F90 are defined as:

```
integer, parameter :: ncnst=4                      ! number of constituents
  type(boundarydata_type) :: chemdata
  character(len=6), dimension(ncnst), parameter :: nc_names = & ! constituent names
       (/'TN2O  ', 'TCH4  ', 'TCFC11', 'TCFC12'/)

  integer :: n2o_rxt, ch4_rxt, cfc11_rxt, cfc12_rxt, lyman_alpha_rxt
  integer :: n2o_ndx, ch4_ndx, cfc11_ndx, cfc12_ndx
  integer :: ghg_ndx(ncnst)
  character(len=6) :: ghg_bnd_names(ncnst)

  logical :: lyman_alpha = .false.
  type(boundarydata_type) :: h2orate_data
  character(len=4), parameter :: h2orate_name = 'jh2o'
```

These fields are specified in the namelist as being in:

bndtvg = '/glade/p/cesmdata/cseg/inputdata/atm/cam/ggas/noaamisc.r8.nc'
h2orates = /glade/p/cesmdata/cseg/inputdata/atm/waccm/phot/xh2o_c080826.nc

ncdump of noaamisc.r8.nc:

```
        double TCH4(time, lat, lev, lon) ;
                TCH4:long_name = "methane loss frequency" ;
                TCH4:units = "1/sec" ;
        double TN2O(time, lat, lev, lon) ;
                TN2O:long_name = "nitrous oxide loss frequency" ;
                TN2O:units = "1/sec" ;
        double TCFC11(time, lat, lev, lon) ;
                TCFC11:long_name = "cfc11 loss frequency" ;
                TCFC11:units = "1/sec" ;
        double TCFC12(time, lat, lev, lon) ;
                TCFC12:long_name = "cfc12 loss frequency" ;
                TCFC12:units = "1/sec" ;
...
// global attributes:
                :Conventions = "NCAR-CSM" ;
                :Source = "NOAA" ;
                :Written_By = "rosinski" ;
                :Date_Written = "Thu May 15 13:04:11 1997" ;
                :Host = "winterpark" ;
                :Command = "makncmisc" ;

```
And of xh2o_c080826:
```
netcdf xh2o_c080826 {
dimensions:
        time = UNLIMITED ; // (12 currently)
        lev = 56 ;
        lat = 36 ;
        lon = 1 ;
variables:
        float jh2o(time, lat, lev, lon) ;
                jh2o:_FillValue = -999.f ;
...
// global attributes:
                :Conventions = "NCAR-CSM" ;
                :Source = "NOAA" ;
                :Written_By = "rosinski" ;
                :Date_Written = "Thu May 15 13:04:11 1997" ;
                :Host = "winterpark" ;
                :Command = "makncmisc" ;
                :history = "Tue Aug 26 09:23:32 2008: ncpdq -a time,lat,lev,lon xh2o_c080712.nc xh2o_c080826.nc\n",
                        "Sat Jul 12 11:57:55 2008: ncks -A -v date,datesec,time,lat,lev,lon,P0 noaamisc.r8.nc xh2o_c080712.nc" ;
                :nco_openmp_thread_number = 1 ;
}

```
The data in the bndtvg file comes from 1997. The h2orates, it is not clear, but at the latest it's from 2008.

In addition, does the water loss rate only include photolysis? This will be practically zero throughout the domain. I would think the loss rate from the reaction with O(1D) would more relevant to getting H2O correct.

**Comment by lkemmons** [comment]
I had forgotten about 'winterpark'!  I started working at NCAR in 1997.

I have not looked at the ghg code module to see what is done there.  Does it do anything that could not be done in the mechanism?
I guess we do not have O1D in the oxidants file, but maybe we should just add it, then everything could be calculated online and not need specified rates.
These are the reactions in TSMLT.  Need to modify the products for this.

[jn2o]                        N2O + hv -> O1D + N2 
[O1D_N2Oa]             O1D + N2O  -> 2*NO                                       ; 7.26e-11, 20 
[O1D_N2Ob]             O1D + N2O  -> N2 + O2                                    ; 4.64e-11, 20 
[CH4_OH]               CH4 + OH  -> CH3O2 + H2O                                 ; 2.45e-12, -1775 
[O1D_CH4a]             O1D + CH4  -> CH3O2 + OH                                 ; 1.31e-10 
[O1D_CH4b]             O1D + CH4  -> CH2O + H + HO2                             ; 3.5e-11 
[O1D_CH4c]             O1D + CH4  -> CH2O + H2                                  ; 9e-12 
[jh2o_b]                      H2O + hv -> H2 + O1D 
[jh2o_c]                      H2O + hv -> 2*H + O 
[jh2o_a]                      H2O + hv -> OH + H 
[O1D_H2O]              O1D + H2O  -> 2*OH                                       ; 1.63e-10, 60 
[jcfcl3]                      CFC11 + hv -> sink 
[jcf2cl2]                     CFC12 + hv -> sink 
[O1D_CFC11]            O1D + CFC11  -> sink                             ; 2.07e-10 
[O1D_CFC12]            O1D + CFC12  -> sink                              ; 1.204e-10

**Reply by dan800** [reply]
IIRC, my first NCAR machine was shavano.

If it's easy to do, then this seems like a better approach - at least there is some temperature dependance in the loss rates. It's worth comparing these calculated rates to the "T" rates above.
Would we assume all CH4 losses produce 2*H2O and sum the a,b,c rate constants? All other products are zero (sink).

We have OH available already, right? So it's just a matter of adding O1D.

**Reply by dan800** [reply]
@lkemmons @tilmes @fvitt Are there existing runs we can use to extract the photorates and O1D? Where does the OH come from that is being used in mam4?

Simplifying Louisa's list, could we use the following?

[jn2o] N2O + hv -> sink
[O1D_N2O] O1D + N2O -> sink ; 1.19e-10, 20
[CH4_OH] CH4 + OH -> 2*H2O ; 2.45e-12, -1775
[O1D_CH4] O1D + CH4 -> 2*H2O ; 1.75e-10
[jh2o] H2O + hv -> sink
[O1D_H2O] O1D + H2O -> sink ; 1.63e-10, 60
[jcfcl3] CFC11 + hv -> sink
[jcf2cl2] CFC12 + hv -> sink
[O1D_CFC11] O1D + CFC11 -> sink ; 2.07e-10
[O1D_CFC12] O1D + CFC12 -> sink ; 1.204e-10

O1D, OH and photolysis rates prescribed (diurnal and monthly averages).

**Reply by lkemmons** [reply]
Photolysis rates can be simulated online each timestep.
Maybe we can just have an O3 -> O1D reaction so we can use the existing specified oxidants.

**Reply by dan800** [reply]
> Photolysis rates can be simulated online each timestep.
> Maybe we can just have an O3 -> O1D reaction so we can use the existing specified oxidants.

That could work as long as we include quenching of O1D by N2 and O2 in the reaction scheme.

**Comment by dkinnison** [comment]
Do we care about the reformation of H2O from HOx in the MLT. E.g., Brasseur and Solomon, Equation 5.105??

d(H2O)/dt + transport + [JH2O + k(O1D)](H2O) = k(OH)(OH) + k(OH)(HO2) + k(OH)H2) + k(H)(HO2)

JH2O = Ja,b,c (H2O)

**Reply by dkinnison** [reply]
typo: should be above... [JH2O + k(O1D)](H2O)

**Reply by dkinnison** [reply]
third try: (JH2O + k(O1D)) (H2O)

**Reply by dan800** [reply]
My hunch is it's a small source but at least the k(OH)(OH) source term could be included, since OH is prescribed.
